protocol-designer/src/form-types.ts

```diff
@@ -507,14 +510,52 @@ export type ReferenceFields =
   | 'dispense_retract_position_reference'
   | 'mix_position_reference'
 
-export type DelayCheckboxFields =
+export type DelayCheckboxBaseFields =
   | 'aspirate_delay_checkbox'
   | 'dispense_delay_checkbox'
-
-export type DelaySecondFields =
+export type DelayCheckboxMoveLiquidFields =
+  | DelayCheckboxBaseFields
+  | 'aspirate_submerge_delay_seconds'
+  | 'aspirate_retract_delay_seconds'
+  | 'dispense_submerge_delay_seconds'
+  | 'dispense_retract_delay_seconds'
+export type DelaySecondsBaseFields =
   | 'aspirate_delay_seconds'
   | 'dispense_delay_seconds'
-
+export type DelaySecondsMoveLiquidFields =
+  | DelaySecondsBaseFields
+  | 'aspirate_submerge_delay_seconds'
+  | 'dispense_submerge_delay_seconds'
+  | 'aspirate_retract_delay_seconds'
+  | 'dispense_retract_delay_seconds'
```

Comment:
nit
maybe aligning with L518-L521's order style?

```suggestion
  | 'aspirate_submerge_delay_seconds'
  | 'aspirate_retract_delay_seconds'
  | 'dispense_submerge_delay_seconds'
  | 'dispense_retract_delay_seconds'
```

Reply:
I will refactor the order in all of these type unions